packages/vx-responsive/src/enhancers/withParentSize.tsx

```diff
@@ -53,6 +53,7 @@ export default function withParentSize<Props extends WithParentSizeProps = {}>(
     componentWillUnmount() {
       if (this.animationFrameID) window.cancelAnimationFrame(this.animationFrameID);
       if (this.resizeObserver) this.resizeObserver.disconnect();
+      if (this.resize) this.resize.cancel();
```

Comment:
ditto re removing the `if (...)` statement since it's defined in the constructor.

also I think these need to be updated to `this.debouncedResize`/`this.debouncedResize.cancel()` because the unwrapped method wouldn't have the `.cancel()` method (nice that we name this differently in every component 😱 )